packages/main/src/components/SplitterElement/index.tsx

```diff
@@ -51,20 +51,49 @@ export interface SplitterElementPropTypes extends CommonProps {
  */
 const SplitterElement = forwardRef((props: SplitterElementPropTypes, ref: RefObject<HTMLDivElement>) => {
   const { children, style, tooltip, className, minSize, size, resizable, ...rest } = props;
+  const [componentRef, splitterElementRef] = useSyncRef(ref);
+  const { vertical, reset } = useContext(SplitterLayoutContext);
+  const [flex, setFlex] = useState(size !== 'auto' ? `0 0 ${size}` : '1 0 min-content');
```

Comment:
`min-content` doesn't work for Safari: https://caniuse.com/mdn-css_properties_flex-basis_min-content